lib/sandbox/sandbox_linux.go

```diff
@@ -29,7 +29,6 @@ func newNetNs() (*NetNs, error) {
 }
 
 func getNetNs(path string) (*NetNs, error) {
-	return &NetNs{}, nil
```

Comment:
This fixup should probably be squashed into the parent commit (which introduced it).